### PR TITLE
docs: fix conda build

### LINKED ISSUE #139: Fix conda build command
Running the `publish-on-anaconda-org.sh` on anaconda leads to error. 

` RuntimeError: The Poetry configuration is invalid:
    - Additional properties are not allowed ('group' was unexpected)`

### PR DESCRIPTION
**Pull request by hollandjg**
## Description

update the conda settings to successfully build version 2

resolves #139 

### Type of change:
- Documentation update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -393,5 +393,20 @@ docs/         # Directory for static pages to be included in the documentation.
...
+## Conda
+
+This directory includes files for packaging the code for `conda`.
+- [`meta.yaml`](meta.yaml) is the `conda` recipe (configuration file)
+- [`publish-on-anaconda-org.sh`](publish-on-anaconda-org.
+sh) is a script which runs the packaging and outputs the package into the `./dist` directory, then uploads the files to [anaconda.org](https://anaconda.org).
+
+To create and publish the `conda` package:
+- Update `./meta.yaml` with any changed dependencies, to match `pyproject.toml` and taking into account the dependencies which are actually available on the anaconda channels we use: defaults and pytorch. Commit these changes.
+- 🐛 Bugfix: While poetry >=1.2 is not available on anaconda.org, delete the [tool.poetry.group...] parts of pyproject.toml. These are recognized as incorrect Poetry configuration in poetry 1.1 and below, and will cause the build to fail. Don't commit these changes.
```
Comment (benwandrew) [inline README.md:411]:
```suggestion
- 🐛 Bugfix: Until poetry >=1.2 is available on anaconda.org (you should check each time we publish until it is), delete the [tool.poetry.group...] parts of pyproject.toml. These are recognized as incorrect Poetry configuration in poetry 1.1 and below, and will cause the build to fail. Don't commit these changes.  
```

Comment (benwandrew) [reply README.md:411]: is this phrasing (and nudge) more clear or confusing to you?

**Review by gtdang — COMMENTED** [review]
I'm not really an expert at dependency configs but it was helpful to see this. Let me know if you just need an approval.

File: README.md (hunk 1 above)
Comment (hollandjg) [reply README.md]: Sounds good to me!

**Review by benwandrew — APPROVED** [review]
terrific! not so bloated :)

**Pull request merged**